quinn-proto/src/connection/streams/recv.rs

```diff
@@ -248,6 +248,27 @@ pub struct DidRead<T> {
     pub(crate) max_data: ShouldTransmit,
 }
 
+impl<T> DidRead<T> {
+    /// Returns `true` iff flow control credit should be issued.
+    pub fn should_issue_flow_control_credit(&self) -> bool {
```

Comment:
This is awfully verbose. Maybe `causes_transmit`? External crates (such as `quinn`) don't need to know *why* there's a transmit needed.

Reply:
OK, I renamed it to `causes_transmit` and adjusted the doc comment.